src/util/nodeMatchers.ts

```diff
@@ -74,7 +74,7 @@ export function chainedMatcher(
  * them in reverse, walking up the ancestor chain from `node`.
  * Returns `null` if any finder in the chain returns null.  For example:
  *
- * ancestorChainNodeFinder(0, patternFinder("foo", "bar"), patternFinder("bongo"))
+ * ancestorChainNodeMatcher([patternFinder("foo", "bar"), patternFinder("bongo")], 0)
```

Comment:
I noticed this while I was poking around the code to find places where we return a removal range 🤷‍♂️